spec.md

```diff
@@ -547,212 +547,134 @@ See [@sec:mem] for more details.
 
 # Types
 
-FIRRTL has four classes of types: _ground_ types, _aggregate_ types, _reference_ types, and _property_ types.
-Ground types are fundamental and are not composed of other types.
-Aggregate types and reference types are composed of one or more aggregate or ground types.
-Reference types may not contain other reference types.
-Property types represent information about the circuit that is not hardware.
+FIRRTL has four classes of types: **ground** types, **aggregate** types, **probe** types, and **property** types.
```

Comment:
you seem to have removed analog from the ground types list below, but not added them to this list... where do they go in this classification?

Reply:
I need to ask some questions about analog types to decide where they should go.

Related to that is what "ground" type even means. The way we classify types is really lumpy, and I want to feel things out a bit.

Reply:
sorry how was this resolved?

Reply:
`Analog` was simply moved back to the Ground Types subsection.